### DISCUSSION TITLE
Changing textcolor taskbar

### DISCUSSION THREAD

**Discussion by winmember** [Open-Shell/Open-Shell-Menu#1295] [General]
Hi.
I tried to change the textcolor in the taskbar but when I change the taskbarcolor into black and the textcolor of the taskbar  into white
only the datecolortext has changed into white .
The rest on the bottomright in the taskbar such as the actioncenter, sound, internet stays black.
Maybe that's because that is not text but some shortcut ?
Is it possible to display these also into white ?
Sorry I cannot add a screenshot.
I use Open-Shell Menu 4.4.188

Thanx.

**Comment by ge0rdi** [comment]
Changing taskbar text color simply doesn't work anymore (I guess for long time already).
It is not possible to add it without substantial changes (that are unlikely to happen).

**Comment by winmember** [comment]
Thanx ge0rdi, clear now.
